### ISSUE TITLE
`org.openrewrite.rewrite` plugin conflicts with `io.moderne.rewrite`

### ISSUE THREAD

**Issue by jkschneider** [openrewrite/rewrite-gradle-plugin#137]
Both try to configure an extension named `rewrite`.

**Comment by shanman190** [comment]
Just FYI, order matters here:

Success:
```
plugins {
    id "org.openrewrite.rewrite" version "latest.release"
    id "io.moderne.rewrite" version "latest release"
}
```

Failure:
```
plugins {
    id "io.moderne.rewrite" version "latest release"
    id "org.openrewrite.rewrite" version "latest.release"
}
```

**Comment by shanman190** [comment]
So the easiest way at the moment would be to just swap the OSS plugin to `TaskContainer#maybeCreate(..)`. That would resolve the issue so that order no longer mattered.

If we were talking about this more longer term, you'd probably have to ask the `TaskContainer` if it knew about an extension named `rewrite`, if none existed register one. If one exists, then apply any configurations.

**Comment by sambsnyd** [comment]
This has been fixed as of 5.34.0
